src/app/service/eco-news/eco-news.service.ts

```diff
@@ -56,12 +56,12 @@ export class EcoNewsService {
       .splice(0, 3);
     this.sortedLastThreeNews.next(separetedNews);
   }
-
+  
```

Comment:
Please revert these changes, 'cause this functionality is out of the scope of your ticket

Reply:
fixed